src/main/java/bio/terra/common/CloudPlatformWrapper.java

```diff
@@ -56,11 +60,23 @@ void ensureValidRegion(CloudPlatform platform,
     public abstract List<? extends StorageResource<?, ?>>
         createStorageResourceValues(DatasetRequestModel datasetRequest);
 
+    public static Stream<? extends StorageResource<?, ?>>
+        createGoogleResourcesForAzure(DatasetRequestModel datasetRequestModel) {
```

Comment:
Looking at this name I'm wondering if these methods should be `create()` or `get()`. To me `get()` is probably better since we aren't creating google resources here. (And the resources aren't "create resources".)